README.md

```diff
@@ -414,6 +414,32 @@ This section outlines all the settings and configurations that you can
 put in Django's `settings.py` to fine tune `post-office`'s behavior.
 
 
+### File Storage
+
+If you want Post Office to use a specific file storage for attachments, make sure to configure a
+storage in the `STORAGES` setting in your project `settings.py`, then specify the storage name in
+the `FILE_STORAGE` key. Post Office will use the `default` storage if this is not specified.
+
+```python
+STORAGES = {
+    "default": {
+        "BACKEND": 'django.core.files.storage.FileSystemStorage',
+    },
+    "staticfiles": {
+        "BACKEND": "django.contrib.staticfiles.storage.StaticFilesStorage",
+    },
+    "post_office": {
+        "BACKEND": 'storages.backends.s3boto3.S3Boto3Storage',
+    },
+}
+
+POST_OFFICE = {
+    ...
+    'FILE_STORAGE': 'post_office',
```

Comment:
Mind iterating on this to make declaration simpler? I'm not sure how complex this would be, but this removes one layer of indirection.

```python
POST_OFFICE = {
    'FILE_STORAGE': 'storages.backends.s3boto3.S3Boto3Storage',
}
```

Reply:
I respectfully disagree. I think those layers of indirection are useful to keep all database, cache, and storage settings in one single location in `settings.py`.

Do you have any specific concerns with respect to this? Is it a performance issue, or something?

There are plenty of modern third party apps (I can give examples upon request) that configure their cache to use by letting users specify the cache by key in `CACHES`, and that works pretty well. I expect it to work equally as well with `STORAGES`.

Unless you have a good reason not to, I'd like to keep it as-is.